js/eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    eny:{
+        es6: true,
+        browser:true,
+        node:true
+    },
+    extends: ["airbnb", 'plugin:jest/recommended', 'jest-enzyme'],
+    plugins:  [
+        'babel',
+        'import',
+        'react',
+    ],
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: "module",
+        ecmaFeatures: {
+            jsx: true,
+        }
+    }
+
+}
```

Comment:
You should keep all of these files in **root** , not in js directory
And where is your package.json?

Reply:
add package.json